### PR TITLE
Add script to copy/anonymize the production database onto staging

### PR DESCRIPTION
**Pull request by maurizi**
## Overview

Companion PR to https://github.com/open-apparel-registry/open-apparel-registry/pull/1948

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines
- [x] This PR is targeted at the correct branch (`develop` vs. `ogr/develop`)
- [x] If this PR applies to both OAR and OGR a companion PR has been created

### CODE REVIEW

File: tools/copy_production_db_to_staging [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --skip-final-snapshot"
+        echo "Will run '$DELETE'"
+        read -n 1 -s -r -p "Press any key to delete $NEW_DATABASE_ID"
+        echo
+        $DELETE
+
+        # Run pg_restore
+        export PGPASSWORD="$STAGING_PGPASSWORD" 
+        PG_RESTORE="pg_restore -h $STAGING_DATABASE_ENDPOINT  -U openapparelregistry -d openapparelregistry --clean --if-exists --no-owner $DUMP_FILE"
```
Comment (jwalgran) [inline tools/copy_production_db_to_staging:148]: For OS Hub the username and database name will be `opensupplyhub` so we should change them here for the `ogr/develop` branch version.

Comment (maurizi) [reply tools/copy_production_db_to_staging:148]: There's also `STAGING_ID` and `PRODUCTION_ID` at the top of the script, and we reference `openapparelregistry` in a few other places as well - do we want to set this up to expect an OS Hub staging _and_ production database (assuming one will exist soon)?

Comment (maurizi) [reply tools/copy_production_db_to_staging]: Updated _all_ references of `openapparelregistry` to `opensupplyhub` in https://github.com/open-apparel-registry/open-apparel-registry/pull/1953/commits/df17bd774c75485fb9784efdb8e4f7345e3ef4b2

Comment (jwalgran) [reply tools/copy_production_db_to_staging]: it does make sense for the ogr/develop version of the script to use opensupplyhub everywhere. In the interim period where we use this script to restore OAR prod to OS Hub staging, we will need to override the variables accordingly. This is fine.

**Pull request merged**